packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -216,7 +215,21 @@ test.each(testCases)(
     const aBalanceFinal = await getBalance(aAddress);
     const bBalanceFinal = await getBalance(bAddress);
 
-    expect(BN.sub(aBalanceFinal, aBalanceInit)).toEqual(aFunding);
-    expect(BN.sub(bBalanceFinal, bBalanceInit)).toEqual(bFunding);
+    expect(BN.sub(aBalanceFinal, aBalanceInit)).toEqual(BN.from(1));
+    expect(BN.sub(bBalanceFinal, bBalanceInit)).toEqual(BN.from(4));
   }
 );
+
+const createAllocation = (aAmount: BigNumberish, bAmount: BigNumberish): Allocation => ({
+  allocationItems: [
+    {
+      destination: participantA.destination,
+      amount: hexZeroPad(BigNumber.from(aAmount).toHexString(), 32),
+    },
+    {
+      destination: participantB.destination,
+      amount: hexZeroPad(BigNumber.from(bAmount).toHexString(), 32),
+    },
+  ],
+  assetHolderAddress: ethAssetHolderAddress,
+});
```

Comment:
This could instead receive the participants as input as well, which would save the refactor that pulled `participantA` and `B` out to the global scope.

Global scope is "generally bad" but it's not clear to me that it reduces readability etc in this file. Is there much potential future where this testing suite is expanded and the reuse, rather than re-instantiation of `A` and `B` here presents problems?

For that matter, how immediately confident are we that reusing A and B in this `test.each` loop produces all of the same behaviour as previously?

Reply:
Good point, I've updated the method to take in the participants in https://github.com/statechannels/statechannels/pull/3582/commits/c7d3908b8d7056ef6af1f2964c0c59e0c1a3f34e